data-prepper-plugins/otel-trace-source/README.md

```diff
@@ -14,9 +14,11 @@ source:
 
 * port(Optional) => An `int` represents the port Otel trace source is running on. Default is ```21890```.
 * request_timeout(Optional) => An `int` represents request timeout in millis. Default is ```10_000```.
-* health_check_service(Optional) => A boolean enables a gRPC health check service under ```grpc.health.v1 / Health / Check```. Default is ```false```.
+* health_check_service(Optional) => A boolean enables health check service. When ```true``` enables a gRPC health check service under ```grpc.health.v1.Health/Check```. Default is ```false```.
+  * ```proto_reflection_service``` should be enabled to use the health check service.
```

Comment:
Is this only for the health check?

If not, I think it may make sense to add a comment to the `health_chech_service` bullet point to the effect of: "In order to use the health check service, you must also enable proto_reflection_service".

Reply:
This is specifically for GRPCHealthCheck service. Below is the health check response with ```proto_reflection_service``` set to ```false``` and ```health_check_service``` set to ```true```.

```
grpcurl -plaintext localhost:21890 grpc.health.v1.Health/Check
Error invoking method "grpc.health.v1.Health/Check": failed to query for service descriptor "grpc.health.v1.Health": server does not support the reflection API

```

I added this ```proto_reflection_service should be enabled to use the health check service.``` as sub bullet point to health_check_service. Do want to reword this or not want this as sub bullet point ?

Reply:
I see how you have this now. I'm fine with keeping the sub-bullet. I'd like to make this README easier to map into the existing opensearch.org documentation. So I think not having a sub-bullet may be more appropriate.

See the reference: https://opensearch.org/docs/latest/clients/data-prepper/data-prepper-reference/